pyTST/core.py

```diff
@@ -8,6 +8,15 @@
 from matplotlib import pyplot
 
 
+def moving_average(y, n=10) :
+    y_padded = np.pad(y, (n//2, n-1-n//2), mode='edge')
+    y_smooth = np.convolve(y_padded, np.ones((n,))/n, mode='valid') 
+    return y_smooth
```

Comment:
As discussed earlier, is it really useful considering self.mean_array already exist.